test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -162,6 +226,17 @@ private static int GetCultureInfoDepth(CultureInfo culture)
             return result;
         }
 
+
+        private TestSink _sink = new TestSink();
```

Comment:
Unfortunatly no, you get "A field initializer cannot reference the non-static field, method or property '_sink'".